course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -1059,6 +1089,22 @@ def test_data(self):
         serializer = ContainedCourseRunsSerializer(instance)
         self.assertDictEqual(serializer.data, instance)
 
+    def test_filter_edx_org_short_name(self):
+        """
+        Filter by edx organization.
```

Comment:
Update docstring and test name. The name and docstring should be specific to scenario

Reply:
Done